tests/primer/primer_tool.py

```diff
@@ -88,6 +100,35 @@ def _handle_prepare_command(self) -> None:
             ) as f:
                 f.write(commit_string)
 
+    def _handle_run_command(self) -> None:
+        if Path(PRIMER_DIRECTORY / f"output_{self.config.type}.txt").exists():
+            os.remove(PRIMER_DIRECTORY / f"output_{self.config.type}.txt")
+
+        packages: dict[str, list[dict[str, str | int]]] = {}
+
+        for package, data in self.packages.items():
+            output = self._lint_package(data)
+            packages[package] = output
+            print(f"Successfully primed {package}.")
+
+        with open(
+            PRIMER_DIRECTORY / f"output_{self.config.type}.txt", "a", encoding="utf-8"
+        ) as f:
+            json.dump(packages, f)
+
+    def _lint_package(self, data: PackageToLint) -> list[dict[str, str | int]]:
+        # We want to test all the code we can
+        enables = ["--enable-all-extensions", "--enable=all"]
+        # Duplicate code takes too long and is relatively safe
+        disables = ["--disable=duplicate-code"]
+        arguments = data.directories + data.pylint_args + enables + disables
+        if data.pylintrc_relpath:
+            arguments += [f"--rcfile={data.pylintrc_relpath}"]
+        output = StringIO()
+        reporter = JSONReporter(output)
+        Run(arguments, reporter=reporter, do_exit=False)
```

Comment:
We don't want to lint every primer package twice, so I think we will need to turn off the python test that checks for crashes, yeah?

Reply:
Yes! Although this still needs some work before we get there. We don't actually prime on `astroid` and `black` doesn't take too long. So for now I think this is fine.